test/prefer-event-target.mjs

```diff
@@ -0,0 +1,32 @@
+import outdent from 'outdent';
+import {getTester} from './utils/test.mjs';
+
+const {test} = getTester(import.meta);
+
+test.snapshot({
+	valid: [
+		'class Foo {}',
+		'class Foo extends OtherClass {}',
+		'class Foo extends EventTarget {}',
+	],
```

Comment:
```suggestion
		'class EventEmitter extends Foo {}',
		'class Foo extends foo.EventEmitter {}',
],
```

Reply:
This is not what I wrote, I mean they should be valid.

Reply:
> This is not what I wrote, I mean they should be valid.

I'm sorry for my misunderstanding, I think I missed 14 lines' array closed braket.